### ISSUE TITLE
Is Early Stopping conditioned on Train Loss? Can we change metrics to (Val) Average Precision?

### ISSUE THREAD

**Issue by AlexanderLavelle** [google/yggdrasil-decision-forests#155]
I have noticed the val loss not decreasing. How is early stopping conditioned?

I was curious as well -- is it possible to change the Early Stopping to be based on a metric besides accuracy (our problem wants to maximize AuPRC)

**Comment by ricor07** [comment]
Hello, I'd be glad to work on this. Can i get the issue assigned? Thanks

**Comment by rstz** [comment]
Early stopping uses the validation loss
